### DISCUSSION TITLE
How would i get group funds?

### DISCUSSION THREAD

**Discussion by JanuZz** [noblox/noblox.js#263] [Q&A] [answered]
I have logged in with my cookie and the `setCookie()` function, then when i call the `getGroup()` function it doesn't return the groups funds, any tips or other solutions on how to get the group funds would be much appreciated

**Comment by Jaxenormus** [comment, marked as answer]
AFAIK there is no method to do it in the official package, but I made a fork with that method, which you can find [here](https://github.com/CalebDelbridge/noblox.js/blob/new-methods/lib/group/getGroupFunds.js). It has the correct docs and typing. I haven't gotten around to making a PR yet

**Reply by JanuZz** [reply]
Its alr i figured it out just ising the regular api and a http request, thanks for the effort tho (-;

**Reply by Jaxenormus** [reply]
Great glad you were able to figure it out 👍
